lib/platform/gitlab/index.ts

```diff
@@ -397,11 +397,8 @@ async function fetchPrList(): Promise<Pr[]> {
   const searchParams = {
     per_page: '100',
   } as any;
-  // istanbul ignore if
-  if (config.ignorePrAuthor) {
-    // https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests
-    // default: `scope=created_by_me`
-    searchParams.scope = 'all';
+  if (`${authorId}` && !config.ignorePrAuthor) {
+    searchParams.author_id = `${authorId}`;
   }
```

Comment:
```suggestion
  if (!config.ignorePrAuthor) {
    searchParams.scope = 'created_by_me';
  }
```
This way we don't need the author id. Would be `scope=created-by-me` before GitLab v11

Reply:
It looks like we're using `author_id` in both the merge request endpoint and the issues endpoint, are you happy for me to replace `author_id` in both requests with `scope=created_by_me` and remove `author_id` entirely from this file?

`scope=created_by_me` is supported in both the merge request endpoint and the issues endpoint:
- https://docs.gitlab.com/12.10/ee/api/merge_requests.html#list-project-merge-requests
- https://docs.gitlab.com/12.10/ee/api/issues.html#list-project-issues

Reply:
sure, go ahead 🙃